yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -217,7 +220,27 @@ const (
 	LEFT JOIN %s tii ON trc.schema_name = tii.schema_name AND trc.table_name = tii.object_name and tii.measurement_type='initial'
 	LEFT JOIN %s tis ON trc.schema_name = tis.schema_name AND trc.table_name = tis.object_name
 	LEFT JOIN %s tcc ON trc.schema_name = tcc.schema_name AND trc.table_name = tcc.object_name
-	LEFT JOIN %s otm ON trc.schema_name = otm.schema_name AND trc.table_name = otm.object_name
+	JOIN %s otm ON trc.schema_name = otm.schema_name AND trc.table_name = otm.object_name
+	WHERE otm.object_type NOT IN ('%s', '%s');`
+
+	InsertTableStatsWithoutIOPS = `INSERT INTO %s (schema_name, object_name, row_count, column_count, reads, writes,reads_per_second, writes_per_second, is_index, object_type, parent_table_name, size_in_bytes)
```

Comment:
If possible, we should try and have a single query(irrespective of whether we have data in those tables or not). I remember we wanted to support the scenario where we have partial data as well. 

It's alright if the IOPS table is empty, LEFT JOIN should take care of that, no?

Reply:
hmm yeah you are right.
I was having some issues because of which the final table was not getting populated, i think that was because of error in other metadata query, let me try this.

Reply:
done